radio/apps/schedules/tests.py

```diff
@@ -21,8 +21,9 @@
 from django.core.exceptions import ValidationError
 from django.test import TestCase
 
-from apps.programmes.models import Programme
-from apps.schedules.models import ScheduleBoard, Schedule, MO, TU, WE, TH, FR, SA, SU
+from apps.programmes.models import Programme, Episode
+from apps.schedules.models import ScheduleBoard, Schedule
+from apps.schedules.models import MO, TU, WE, TH, FR, SA, SU
```

Comment:
Extract weekday import to underline these are different things to import (classes vs. constants)